Source/FieldSolver/FiniteDifferenceSolver/EvolveE.cpp

```diff
@@ -202,7 +202,7 @@ void FiniteDifferenceSolver::EvolveECylindrical (
         // Loop over the cells and update the fields
         amrex::ParallelFor(ter, tet, tez,
 
-            [=] AMREX_GPU_DEVICE (int i, int j, int k){
+            [=] AMREX_GPU_DEVICE (int i, int j, int /*k*/){
```

Comment:
@lucafedeli88 @RemiLehe this looks a bit suspicious to me. Are we sure that the index space of `k` is a single value?
Otherwise we should `return` for all but one index value in `k`, otherwise we duplicate work.

Reply:
@ax3l  This is ok. Looking at how the ParalellFor is implemented in 2D, the loop over k will always only be 0.

Reply:
@dpgrote thanks, so the boxes and extents we put into those here are all of size 1 in that index. Thanks for double-checking! :)